### PR TITLE
DOCKER-260 Enable Ubuntu Pro attach

### PR DESCRIPTION
**Pull request by zsoltbalogh**

### CODE REVIEW

File: templates/base/resources/usr/local/bin/update_ubuntu.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+echo "Updating Ubuntu."
+apt-get update
+apt-get upgrade --yes
+apt-get clean
```
Comment (natocesarrego) [inline templates/base/resources/usr/local/bin/update_ubuntu.sh:6]: @zsoltbalogh, why did you decide to have a separate script just to update Ubuntu? Can't you move this code to the script `attach_ubuntu_pro_subscription.sh`?

File: templates/base/resources/usr/local/bin/attach_ubuntu_pro_subscription.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+if [ -n "${1}" ]
+then
+	LIFERAY_DOCKER_UBUNTU_PRO_TOKEN="${1}"
```
Comment (natocesarrego) [inline templates/base/resources/usr/local/bin/attach_ubuntu_pro_subscription.sh:5]: @zsoltbalogh, where do you plan to set this info? Is it intended to our customers use this variable?

File: templates/base/resources/usr/local/bin/attach_ubuntu_pro_subscription.sh [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+if (pro status 2>/dev/null | grep -q "Subscription: Ubuntu Pro" 2>/dev/null)
+then
+	echo "Ubuntu Pro subscription is already active."
+
+	exit 0
+fi
+
+apt-get update && \
+DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install --no-install-recommends --yes ubuntu-advantage-tools && \
+pro attach "${LIFERAY_DOCKER_UBUNTU_PRO_TOKEN}"
```
Comment (natocesarrego) [inline templates/base/resources/usr/local/bin/attach_ubuntu_pro_subscription.sh:24]: @zsoltbalogh, shouldn't we use the same pattern BChan uses to build the scripts? I mean, have a function named `main` which is the entrypoint of the script? For example: https://github.com/liferay/liferay-docker/blob/d735f8d188b71c72be90cfc57a5951179417eadb/templates/bundle/resources/usr/local/bin/configure_liferay.sh#L104

File: templates/base/resources/usr/local/bin/update_ubuntu.sh (hunk 1 above)
Comment (zsoltbalogh) [reply templates/base/resources/usr/local/bin/update_ubuntu.sh:6]: Please check the ticket, you can see that we should still recommend updating Ubuntu and this is why the script is separated. It was initially together, but when I wrote the docs, I separated it.

File: templates/base/resources/usr/local/bin/attach_ubuntu_pro_subscription.sh (hunk 2 above)
Comment (zsoltbalogh) [reply templates/base/resources/usr/local/bin/attach_ubuntu_pro_subscription.sh:5]: Please check the ticket description. The customer would bring it.

**Review by zsoltbalogh — COMMENTED** [review]
I have responded to the questions I'm not sure if I should submit now?

File: templates/base/resources/usr/local/bin/update_ubuntu.sh [hunk 4]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+echo "Updating Ubuntu."
+apt-get update
```
Comment (natocesarrego) [inline templates/base/resources/usr/local/bin/update_ubuntu.sh:4]: @zsoltbalogh, wouldn't be good separate these lines by a line break?

**Comment by natocesarrego** [comment]
Thanks for your clarification, @zsoltbalogh! :smile: 

From my point of view, my only SF suggestion is this [one](https://github.com/natocesarrego/liferay-docker/pull/206/commits/89ed061e690d6029234b08c891004d82e3d3b4ff#r1765677272) regarding this pull. :slightly_smiling_face: 

Feel free to forward this pull to Brian! :smile:

**Pull request closed without merging**